CriticalMass/IdentifiableAnnnotation.swift

```diff
@@ -7,25 +7,20 @@
 
 import MapKit
 
-class IdentifiableAnnnotation: MKPointAnnotation {
-    enum UserType: Int, Decodable {
-        case friend
-        case user
-    }
-
-    var type: UserType = .user
+class IdentifiableAnnnotation<T>: MKPointAnnotation {
```

Comment:
Could this be just MKAnnotation instead to make it more versatile for other annotation types for example

Reply:
Not sure if that's possible. MKAnnotation is just a protocol and not a class. When I use MKAnnotation the annotation don't seem to update anymore

Reply:
🤔 I would assume it would still work since Confirming to the protocol would require to add the coordinate property but it is there already, right?

Reply:
Maybe you need to define the coordinate property as `@objc dynamic` like in https://developer.apple.com/documentation/mapkit/mapkit_annotations/annotating_a_map_with_custom_data